modules/test-utils/src/browser-automation/browser-test-driver.ts

```diff
@@ -80,6 +82,11 @@ export default class BrowserTestDriver extends BrowserDriver {
       }
 
       const result = await this._openPage(url, config);
+
+      if (config.onFinish) {
+        await config.onFinish(this.failures === 0);
```

Comment:
Nit: boolean values are a bit iffy. 
- Should we pass in an object with keys to make it clear what things mean? 
- Also pass in the test driver instance per comment above?